homeassistant/components/dlna_dmr/config_flow.py

```diff
@@ -470,4 +470,20 @@ def _is_ignored_device(discovery_info: Mapping[str, Any]) -> bool:
     if discovery_info.get(ssdp.ATTR_UPNP_DEVICE_TYPE) not in DmrDevice.DEVICE_TYPES:
         return True
 
+    # Special cases for devices with other discovery methods (e.g. mDNS), or
+    # that advertise multiple unrelated (sent in separate discovery packets)
+    # UPnP devices.
+    manufacturer = discovery_info.get(ssdp.ATTR_UPNP_MANUFACTURER, "").lower()
+    model = discovery_info.get(ssdp.ATTR_UPNP_MODEL_NAME, "").lower()
+
+    if manufacturer.startswith("xbmc") or model == "kodi":
+        # kodi
+        return True
+    if manufacturer.startswith("samsung") and "tv" in model:
+        # samsungtv
+        return True
+    if manufacturer.startswith("lg") and "tv" in model:
+        # webostv
+        return True
```

Comment:
Should this be a tuple or something that is iterated, instead of multiple separate if statements?

Reply:
Possibly, but given the pattern was already broken between Kodi and the two TVs, I wasn't sure how similar future additions would be.

If you'd prefer it as an iterated check, I can make it work (the `kodi` check would be made to look like the others, but would still detect it).

Reply:
Oh right, there is already a slight difference. 👍